### ISSUE TITLE
Add more names for public_bookcase

### ISSUE THREAD

**Issue by Coehill** [openstreetmap/id-tagging-schema#1332]
They are commonly referred to as "little free libraries" near me, so let's include this an a naming option.

**Comment by danieldegroot2** [comment]
This and similar terms/brands should already be covered by the term 'library'. The 'Little Free Library' brand itself is already covered by the [name-suggestion-index](https://github.com/osmlab/name-suggestion-index).

**Comment by Coehill** [comment]
@Zverik Do you know why Little Free Library is appearing under the Micromapping Mode of EveryDoor whereas Little Free Pantry is showing under Amenity Mode? IMO, Little Free Library should be under Amenity Mode.

**Comment by mnalis** [comment]
@Coehill you should better ask Every Door questions in [their GitHub](https://github.com/Zverik/every_door) (or it is likely to be missed/forgotten, and also needlessly creating noise in this repo which can't help solving it). 

But (since I've already added to the noise) AFAIR @Zverik [previously stated](https://github.com/streetcomplete/StreetComplete/issues/5152#issuecomment-1707118120) that "Amenity" in Every Door should be:

> [...] venue manned by people or providing paid services. So, shops, churches, vending machines and parcel delivery machines, but not post boxes or garbage bins.

`amenity=public_bookcase` is explicitly (so likely intentionally) excluded by being mentioned in [wrongAmenities](https://github.com/Zverik/every_door/blob/b29901b67eb94dc0bbde60931bb4306b5a6ebf60/lib/helpers/good_tags.dart#L147) in Every Door. Whether `amenity=food_sharing` should be excluded too (or `public_bookcase` removed from `wrongAmenities`) is a question which should be [asked there](https://github.com/Zverik/every_door/issues) (if you still think it needs  correcting - to me it looks OK-ish as it is)

**Comment by tordans** [comment]
Is there something we can do on the id-tagging-schema side of things? If so, could you summarize that, please?

I would prefer to close this issue otherwise to keep us focussed.

**Comment by Zverik** [comment]
Regarding the topic, I'd say the pantry should also go to the micromapping mode. Maybe I just forgot to add it. Otherwise, I'm pro merging the pull request and closing this ticket.
